src/jit/morph.cpp

```diff
@@ -17393,7 +17392,7 @@ Compiler::fgWalkResult Compiler::fgMorphStructField(GenTree* tree, fgWalkData* f
                 if (verbose)
                 {
                     printf("Replacing the field in promoted struct with a local var:\n");
-                    fgWalkPre->printModified = true;
+                    // fgWalkPre->printModified = true;
```

Comment:
`GenTreeVisitor` doesn't have this facility. I'll see if it can be replaced with something else.